### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by svitalis123**
## Awesome books: a complete website with navigation
**In this project I accomplished**
- We created a new branch.
- We Added Three sections (Booklist, Add book, Contact).

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -21,51 +39,127 @@ class Awesomebooks {
...
+//     }
+//   }
+
+//   dt = dt.replace(ot, mt);
+//   return dt;
+// }
+
+// function sliceDatetime(dt) {
+//   return dt.slice(0, 16);
+// }
```
Comment (youmari) [inline js/main.js]: - [ ] it would be better if you remove the unused commented code in order to keep your file clean and pro

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @svitalis123 , @devMunyi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ PRO PR
✔️ Github flow 
✔️ No linter error 

## Required Changes ♻️

- [ ] this is a paid programming project so it would be better if both of you write code in this PR

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (svitalis123) [reply js/main.js]: Tank you sir

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi team, 

Kudos on deploying the project!!!

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**